lib/fastlane/plugin/wpmreleasetoolkit/helper/release_notes_helper.rb

```diff
@@ -11,7 +11,13 @@ def self.add_new_section(path:, section_title:)
 
         # Find the index of the first non-empty line that is also NOT a comment.
         # That way we keep commment headers as the very top of the file
-        line_idx = lines.find_index { |l| !l.start_with?('***') && !l.start_with?('//') && !l.chomp.empty? }
+        line_idx = lines.find_index do |l|
+          !l.start_with?('***') &&
+            !l.start_with?('//') &&
+            !l.start_with?('#') &&
+            !l.start_with?('- ') &&
+            !l.chomp.empty?
```

Comment:
Rubocop says this formatting is fine, but it seems strange to have the last four lines indented more. 

I broke them into separate lines for readability. Is there a better way of doing it?

Reply:
I think the additional indentation is because the subsequent lines are a continuation of the first line so I agree it can look strange, but it's expected at least.

Another option would be to extract this condition in a function to make this line shorter and use an array of comment prefixes. Something like:

```Ruby
def comment?(line)
  ['***', '//', '#', '- '].any? { |s| line.start_with?(s) }
end
```
Then we could go back to write this code as a one-liner:
```ruby
line_idx = lines.find_index { |l| !is_comment?(l) && !l.chomp.empty? }
```